### ISSUE TITLE
intel iGPUs

### ISSUE THREAD

**Issue by user7z** [zelikos/davincibox#71]
i used the manual installations methode it was simple and succesful but davinci resolve wont start , see the screenshot

**Expected behavior**
i think the container dependencies for intel need to looking at it , and my host my lack a driver that might needed by the containner.

**Screenshots**
![Screenshot_20240519_030108](https://github.com/zelikos/davincibox/assets/161214583/9806917b-f746-495b-a913-36528a328cb7)

**Desktop plasma-meta
 - Distro: Arch Linux
 - GPU: intel iris xe of core i5 1235-U(Alder Lake-U)
 - drivers : vulkan-intel vpl-gpu-rt  intel-media-driver mesa libva

**Comment by zelikos** [comment]
Try running Resolve with `distrobox enter -n davincibox -- /usr/bin/run-davinci rusticl`, and report back.

**Comment by user7z** [comment]
here is screenshots : 
![Screenshot_20240519_045536](https://github.com/zelikos/davincibox/assets/161214583/8e8d107d-83ac-45e3-ae4e-8bdc757796b8)
![Screenshot_20240519_045558](https://github.com/zelikos/davincibox/assets/161214583/027e16d5-42c5-4b17-9c5d-92bf4b23e61d)

and the log file : 
[log-conf-its_a_.xml_file.txt](https://github.com/zelikos/davincibox/files/15370148/log-conf-its_a_.xml_file.txt)

**Comment by zelikos** [comment]
Those screenshots and log aren't very helpful by themselves. Did you get the same error as in your first post, or no? Are you able to edit videos when running Resolve with that command?

**Comment by user7z** [comment]
yes, you see davinci resolve is opened now but i have problem with mp4 doesnot show up to import , and : 
![Screenshot_20240519_052539](https://github.com/zelikos/davincibox/assets/161214583/0f62ae68-4e75-4030-b98c-45db48e97b03)
![Screenshot_20240519_052610](https://github.com/zelikos/davincibox/assets/161214583/1e7fbcab-e54a-4e45-a4fa-af4519803d66)

**Comment by zelikos** [comment]
> i have problem with mp4 doesnot show up to import

That's a limitation of DaVinci Resolve on Linux, not related to davincibox

As for the error in the screenshot, we'll have to wait and see if driver updates improve things

**Comment by user7z** [comment]
the containner contains the essential driver and stuff right ?

**Comment by zelikos** [comment]
Yes, I'm referring to the OpenCL driver that the container ships

**Comment by juliansko** [comment]
installing intel-compute-runtime (via dnf install intel-compute-runtime) worked for me. That's also what the AUR package does afaik

**Comment by smunaut** [comment]
Not sure if this is of any interest but I have an experimental branch that fixes it for intel Arc and newer Xe iGPU : https://github.com/smunaut/compute-runtime/tree/fix-arc

**Comment by zelikos** [comment]
Closing this as we do now ship both `rusticl` and `intel-compute-runtime` as options for Intel GPUs

**Comment by smunaut** [comment]
When using `intel-compute-runtime`, it's important to `export NEOReadDebugKeys=1 DisableDeepBind=1` before running davinci or you might experience random crashes.
